### PR TITLE
Adds skeleton component for App of the week cluster.

### PR DESCRIPTION
**Pull request by ayoadekunle**

### CODE REVIEW

File: client/src/Clusters/AppOfTheWeek/AOTWStyle.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.aotw{
```
Comment (ajwang0) [inline client/src/Clusters/AppOfTheWeek/AOTWStyle.css]: Can you name this file AppOfTheWeekStyle.css instead. I had no idea what "aotw" was for the longest time

Comment (ajwang0) [inline client/src/Clusters/AppOfTheWeek/AOTWStyle.css]: Add a space after aotw

`.atow {`

Comment (ajwang0) [inline client/src/Clusters/AppOfTheWeek/AOTWStyle.css]: Also let's not use the abbreviation for readability

File: client/src/Clusters/AppOfTheWeek/AOTWStyle.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    background-image: 
+        linear-gradient(to right, #C4E9DE 1%, #F0F9F700), 
+            url(./Images/background-image.jpg);
+    text-align: left;
+    margin: 100px;
```
Comment (ajwang0) [inline client/src/Clusters/AppOfTheWeek/AOTWStyle.css]: Why margin 100px? Seems a bit arbitrary

File: client/src/Clusters/AppOfTheWeek/AppOfTheWeek.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import '../styles.css';
+import './AOTWStyle.css'
+import {aotw} from './mockData.js';
+
+class AppOfTheWeek extends Component {
+    render(){
+        return(
+            <div className="aotw">
+                <div className="text">
+                    <h1 className="aotw-header">{aotw.title}</h1>
```
Comment (ajwang0) [inline client/src/Clusters/AppOfTheWeek/AppOfTheWeek.js]: Let's use an `h2` instead of `h1`

**Review by ajwang0 — CHANGES_REQUESTED** [review]

File: client/src/Clusters/AppOfTheWeek/AOTWStyle.css (hunk 2 above)
Comment (ayoadekunle) [reply client/src/Clusters/AppOfTheWeek/AOTWStyle.css]: It is, i kinda needed it to look like the mock

File: client/src/Clusters/AppOfTheWeek/mockData.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import stravaIcon from './Images/strava-icon.jpg';
+import backgroundImage from './Images/background-image.jpg';
+
+export const aotw = {
+    title: "Wear OS app of the week",
```
Comment (ajwang0) [inline client/src/Clusters/AppOfTheWeek/mockData.js]: Let's just keep this as "App of the week" and omit Wear OS

**Review by ajwang0 — CHANGES_REQUESTED** [review]

**Review by ajwang0 — DISMISSED** [review]

**Review by ajwang0 — APPROVED** [review]

**Pull request merged**